src/checkers/inference/model/CombineConstraint.java

```diff
@@ -38,6 +33,34 @@ public Slot getResult() {
         return result;
     }
 
+    @Override
+    public <S, T> T serialize(Serializer<S, T> serializer) {
+        return serializer.serialize(this);
+    }
+
+    // First = Target, Second = Declared, Third = Result
+    // See ConstraintEncoderCoordinator
+
+    @Override
```

Comment:
Same question with `TernaryConstraint`.

Reply:
As commented before. Not changing in this PR.